src/test/java/stepDefinition/TestAndreiCiuvagaSteps.java

```diff
@@ -0,0 +1,88 @@
+package stepDefinition;
+
+import cucumber.api.DataTable;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import pages.HomePage;
+import pages.PageObject;
+
+import java.util.List;
+
+public class TestAndreiCiuvagaSteps extends PageObject {
+    @Given("^User is on 'HomePage'$")
+    public void userIsOnHomePage() {
+    }
```

Comment:
Why is this step empty ?

Reply:
Hi!
We use @before for every scenario so when we execute this piece of code we are get landed on Homepage, so I put Given in Background as is common for every scenario, we not doing any action here, so I left it empty